src/main/java/it/pagopa/transactions/commands/handlers/TransactionActivateHandler.java

```diff
@@ -93,13 +92,13 @@ public TransactionActivateHandler(
     }
 
     public Mono<Tuple2<Mono<TransactionActivatedEvent>, String>> handle(
-                                                                        TransactionActivateCommand command
+                                                                        Tuple2<TransactionActivateCommand, TransactionId> input
```

Comment:
`TransactionActivateCommand` is a purely internal data structure, I would add a `transactionId` field there instead of using a tuple

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/289/commits/88eb6998da70004e6f5c3ab259e9a30b6e430614